lib/lita/handlers/reviewme.rb

```diff
@@ -96,16 +96,16 @@ def comment_on_github(response)
         end
       end
 
-      def mention_reviewer(response)
+      def mention_reviewer(response, room: get_room(response))
         url = response.matches.flatten.first
-        reviewer = next_reviewer
+        reviewer = next_reviewer(room)
         response.reply(chat_mention(reviewer, url))
       end
 
       private
 
-      def next_reviewer
-        redis.rpoplpush(REDIS_LIST, REDIS_LIST)
+      def next_reviewer(room)
+        ns_redis(room).rpoplpush(REDIS_LIST, REDIS_LIST)
```

Comment:
That makes sense, I was afraid that room is an object and it would serialize the full object.